app/components/AccountPlan/AccountPlan.tsx

```diff
@@ -40,67 +43,71 @@ const EnterpriseList = () => {
       <List.Item>Custom relays per day</List.Item>
       <List.Item>Custom volume plans</List.Item>
       <List.Item>No throughput limit</List.Item>
-      {/*<List.Item>5 Applications</List.Item>*/}
-      {/*<List.Item>10 Team Members</List.Item>*/}
       <List.Item>Direct Customer support</List.Item>
       <List.Item>Access all supported chains</List.Item>
       <List.Item>Global region support</List.Item>
       <List.Item>ETH Trace supported</List.Item>
     </List>
   )
 }
+*/
+const UnlimitedList = () => {
+  return (
+    <List center icon={<LuCheck size="18px" />} size="sm" spacing="xl">
+      <List.Item>Unlimited relays per month</List.Item>
+      <List.Item>100,000 relays free per month</List.Item>
```

Comment:
Just wondering if we want to change how the stripe worker handles this? Currently it doesn't really give 100,000 free relays, it just doesn't charge for the first million until the user hits 100,001 free relays.

eg. >= 100,000 relays = $0
100,001 - 1,000,000 relays = $2
1,000,001 relays = $4, etc.

Is that correct? If so maybe we should reword this. If not I'll need to make a change to the Stripe worker.

Reply:
No, the logic in the stripe worker is correct. I'll think about how to reword this.